bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/types/BType.java

```diff
@@ -133,11 +133,16 @@ public int hashCode() {
     }
 
     public String getName() {
-        return typeName;
+        return typeName == null ? "" : typeName;
```

Comment:
Need to decide if we should return null instead of "" if `typeName` is null.

Reply:
Can typeName can be null? Is that a bug?

Reply:
We seem to have explicitly passed null in quite a few places, e.g., io.ballerina.runtime.internal.types.BTupleType#BTupleType(java.util.List<io.ballerina.runtime.api.types.Type>)